pkg/seccomp/default_linux.go

```diff
@@ -382,6 +383,7 @@ func DefaultProfile() *Seccomp {
 				"utimensat_time64",
 				"utimes",
 				"vfork",
+				"vmsplice",
```

Comment:
in what cases is `vmsplice` needed?

It was dropped on purpose with 7ced5daafa0e36102eb931050ba3ff99f42bdfac as it is an unsafe syscall

Reply:
It is being called and causes a failure, when running rootless podman within rootless podman.  I will show you on Monday.

Reply:
@giuseppe Please try to run

$ podman run --security-opt label=disable --user podman --device /dev/fuse quay.io/rhatdan/podman podman run alpine echo hello